src/index.html

```diff
@@ -4,9 +4,43 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/styles.css">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__row">
+        <div class="calendar__row-block calendar__row-block--sun"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+        <div class="calendar__row-block"></div>
+      </div>
+    </div>
```

Comment:
You should have one div for calendar, and 31 divs inside for each day 
`calendar__row` is unnecessary. Also both modifiers should be on `calendar` and not on individual day.
Also change your classes to match readme and checklist.